Data/ByteString/Short/Internal.hs

```diff
@@ -630,3 +1642,55 @@ moduleErrorIO fun msg = throwIO . userError $ moduleErrorMsg fun msg
 
 moduleErrorMsg :: String -> String -> String
 moduleErrorMsg fun msg = "Data.ByteString.Short." ++ fun ++ ':':' ':msg
+
+
+-- Find from the end of the string using predicate
+findFromEndUntil :: (Word8 -> Bool) -> ShortByteString -> Int
```

Comment:
Please document what this function returns when the predicate is false for the entire SBS.